edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Mqtt/StringIdentity.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Hub.Mqtt
+{
+    using Microsoft.Azure.Devices.Edge.Hub.Core.Identity;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Azure.Devices.ProtocolGateway.Identity;
+
+    public class StringIdentity : IDeviceIdentity
```

Comment:
nit: rename to SimpleIdentity or AuthenticatedIdentity or even DeviceIdentity

Reply:
renamed